### PR TITLE
fix: lint error

### PR DESCRIPTION
**Pull request by elioist**
[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [x] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

### CODE REVIEW

File: components/_util/throttleByAnimationFrame.ts [hunk 1]
```diff
@@ -15,7 +15,7 @@ function throttleByAnimationFrame<T extends any[]>(fn: (...args: T) => void) {
   };
 
   throttled.cancel = () => {
-    raf.cancel(requestId!);
+    requestId && raf.cancel(requestId);
```
Comment (afc163) [inline components/_util/throttleByAnimationFrame.ts:18]: requestId 有没有可能是 0？

Comment (li-jia-nan) [reply components/_util/throttleByAnimationFrame.ts:18]: 不等价了

Comment (cullyfung) [reply components/_util/throttleByAnimationFrame.ts]: 查了下mdn，返回值是一个是个非零值

**Comment by afc163** [comment]
eslint 挂了

**Comment by afc163** [comment]
master 分支上 eslint 没有报错，所以原始问题是啥？

**Comment by cullyfung** [comment]
> master 分支上 eslint 没有报错，所以原始问题是啥？

是因为是迁移到biome的原因吗，我改这个的原因也是因为biome的lint校验

**Comment by afc163** [comment]
> master 分支上 eslint 没有报错，所以原始问题是啥？

ping~

**Comment by afc163** [comment]
先关了，遇到 lint error 了再开吧。

**Pull request closed without merging**